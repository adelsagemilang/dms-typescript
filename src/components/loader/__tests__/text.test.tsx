import React from 'react';
import { shallow } from 'enzyme';
import TextLoaders from '../text';

describe('Text Loader tests', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TextLoaders height={400} width={400} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders correctly while props changed', () => {
    const wrapper = shallow(<TextLoaders height={400} width={400} />);
    expect(wrapper).toMatchSnapshot();

    wrapper.setProps({
      height: 300,
      width: 300,
    });
    expect(wrapper).toMatchSnapshot();
  });
});
