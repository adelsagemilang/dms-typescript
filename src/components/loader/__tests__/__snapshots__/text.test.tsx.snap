// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Loader tests renders correctly 1`] = `
<ContentLoader
  height={400}
  speed={2}
  width={400}
>
  <rect
    height={400}
    rx="0"
    ry="0"
    width={400}
    x="0"
    y="0"
  />
</ContentLoader>
`;

exports[`Text Loader tests renders correctly while props changed 1`] = `
<ContentLoader
  height={400}
  speed={2}
  width={400}
>
  <rect
    height={400}
    rx="0"
    ry="0"
    width={400}
    x="0"
    y="0"
  />
</ContentLoader>
`;

exports[`Text Loader tests renders correctly while props changed 2`] = `
<ContentLoader
  height={300}
  speed={2}
  width={300}
>
  <rect
    height={300}
    rx="0"
    ry="0"
    width={300}
    x="0"
    y="0"
  />
</ContentLoader>
`;
